src/Whoops/Util/Misc.php

```diff
@@ -41,4 +41,21 @@ public static function translateErrorCode($error_code)
         }
         return "E_UNKNOWN";
     }
+    
+    /**
+     * Determine if an error level is fatal (halts execution)
+     * 
+     * @param int $level
+     * @return boolean
```

Comment:
[Can be any](http://www.phpdoc.org/docs/latest/references/phpdoc/types.html#abnf), though.